src/PrestaShopBundle/Install/Install.php

```diff
@@ -311,22 +311,18 @@ public function installDatabase($clear_database = false)
             return false;
         }
 
-        return $this->generateSf2ProductionEnv();
+        return $this->generateSf2Env();
     }
 
     /**
-     * Pass SF2 to production
      * cache:clear
      * assetic:dump
      * doctrine:schema:update.
      *
      * @return bool
      */
-    public function generateSf2ProductionEnv()
+    public function generateSf2Env()
     {
-        if (defined('_PS_IN_TEST_')) {
-            return true;
-        }
         $schemaUpgrade = new UpgradeDatabase();
         $schemaUpgrade->addDoctrineSchemaUpdate();
```

Comment:
if you check inside this method it runs the command with --env=PS_ENV, so it should work as expected in TEST env too.